parsers/IL.py

```diff
@@ -1,43 +1,76 @@
-#!/usr/bin/env python3
-# coding=utf-8
+"""
+Israel's electricity system load parser.
 
-# This parser returns Israel's electricity system load (assumed to be equal to electricity production)
-# Source: Israel Electric Corporation
-# URL: https://www.iec.co.il/en/pages/default.aspx
-# Shares of Electricity production in 2018: 65.6% oil, 34.4% gas (source: IEA; https://www.iea.org/data-and-statistics?country=ISRAEL&fuel=Electricity%20and%20heat&indicator=Electricity%20generation%20by%20source)
+Source: Israel Electric Corporation
+URL: https://www.iec.co.il/en/pages/default.aspx
 
+Shares of Electricity production in 2019:
+    64.0% Gas
+    31.0% Coal
+    1.0% oil,
+    4.0% Renewables
+    (source: IEA; https://www.iea.org/data-and-statistics?country=ISRAEL&fuel=Electricity%20and%20heat&indicator=Electricity%20generation%20by%20source)
+"""
 
-import arrow
-import requests
 import re
+
+import arrow
+from requests import get
 from bs4 import BeautifulSoup
 
+IEC_URL = "www.iec.co.il"
+IEC_PRODUCTION = (
+    "https://www.iec.co.il/_layouts/iec/applicationpages/lackmanagment.aspx"
+)
+PRICE = "50.66"  # Price is determined yearly
```

Comment:
Maybe a check for the current year somewhere would be a good idea, in case nobody remembers to update the price for 2022?

Reply:
I can probably scrape the site while I'm at it, consider this a placeholder.